packages/packages.csv

```diff
@@ -106,14 +107,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
 Microsoft.AspNetCore.StaticFiles,ship
 Microsoft.AspNetCore.TestHost,ship
 Microsoft.AspNetCore.Testing,noship
-Microsoft.AspNetCore.WebSockets.Client,ship
+Microsoft.AspNetCore.WebSockets.Client,noship
 Microsoft.AspNetCore.WebSockets.Protocol,ship
 Microsoft.AspNetCore.WebSockets.Server,ship
 Microsoft.AspNetCore.WebUtilities,ship
 Microsoft.Cci,ext
 Microsoft.CodeAnalysis.*,ext
 Microsoft.CSharp,ext
-Microsoft.Data.Sqlite,ship
+Microsoft.Data.Sqlite,ship-ext
```

Comment:
Actually, I don't think this is right. It's more nuanced - or perhaps we should split apart this package into two, like we do with libuv. The problem here is that now we don't sign *anything* in this package, but that is incorrect.

This package contains two sets of assets:
* Microsoft.Data.Sqlite.dll, which we wrote, and should be signed
* sqlite.dll, which we did not write, and should not be signed

This list doesn't accurately reflect that split.

I recommend splitting this into two packages, where we sign the contents of one, and not the other.